### PR TITLE
Dawnified Article sliders, Few tweaks for sldr

### PR DESCRIPTION
**Pull request by chrisdavery**
Preview Customizer: https://nodemcu-smartthings.myshopify.com/admin/themes/149348319547/editor?section=template--18965856289083__two-col-carousel&category=gid%3A%2F%2Fshopify%2FOnlineStoreThemeSettingsCategory%2FColors%3Ftheme_id%3D149348319547%26first_setting_id%3Dcolors_solid_button_labels

Web Preview: https://konnected.io/?preview_theme_id=149348319547

### CODE REVIEW

File: sections/k-two-col-carousel.liquid [hunk 1]
```diff
@@ -1,55 +1,242 @@
 <link rel="stylesheet" href="{{ 'k-two-col-carousel-styles.css' | asset_url }}" media="print" onload="this.media='all'">
```
Comment (heythisisnate) [inline sections/k-two-col-carousel.liquid:1]: What about this stylesheet? It seems like it (mostly?) won't be needed anymore? or do these styles still apply?

**Review by heythisisnate — COMMENTED** [review]
this is excellent! Just had one question about the old stylesheet to clarify, otherwise this is good to go.

File: sections/k-two-col-carousel.liquid (hunk 1 above)
Comment (chrisdavery) [reply sections/k-two-col-carousel.liquid:1]: no, we need it, applied new css codes on that file

Comment (chrisdavery) [reply sections/k-two-col-carousel.liquid:1]: makin new commit on this branch

Comment (chrisdavery) [reply sections/k-two-col-carousel.liquid:1]: @heythisisnate made a new commit and edits on this branch. Please check

**Comment by heythisisnate** [comment]
@chrisdavery the code looks great. However the Merge PR from main shouldn't be in there. We should not be making PRs and merging into feature branches. I will show you the process to rebase your feature branch when the main branch gets ahead (next week).

In the meantime I'll remove the merge commit and update this branch.

**Pull request merged**